ckan/cli/server.py

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+import os
+
+import click
+from flask import Flask, current_app
+from werkzeug.serving import run_simple
+
+from ckan.cli import load_config, click_config_option
+
+
+@click.command(u'run', short_help=u'Start development server')
```

Comment:
is `run` a common way to start a flask server or did you just prefer `ckan run` over `ckan serve`?

Reply:
Flask is using the command `run` for starting the app, i believe its a good practice to follow up the flask way thus i choose `run` over `serve`.